README.md

```diff
@@ -66,12 +66,18 @@ Using the `-s/--source`, `-t/--target` and `-o/--output` arguments will run the
 Disclaimer
 ----------
 
-This software is meant to be a productive contribution to the rapidly growing AI-generated media industry. It will help artists with tasks such as animating a custom character or using the character as a model for clothing etc.
+This software is meant to be a productive contribution to the rapidly growing AI-generated media industry. It will help artists with tasks such as:
+- Animating a custom character
+- Using the character as a model for clothing and more.
 
-The developers of this software are aware of its possible unethical applications and are committed to take preventative measures against them. It has a built-in check which prevents the program from working on inappropriate media including but not limited to nudity, graphic content, sensitive material such as war footage etc. We will continue to develop this project in the positive direction while adhering to law and ethics. This project may be shut down or include watermarks on the output if requested by law.
+The developers of this software are aware of its possible unethical applications and are committed to taking preventative measures against them. It includes a built-in check that prevents the program from working on inappropriate media, including but not limited to:
+- Nudity
+- Graphic content
+- Sensitive material like war footage, etc.
 
-Users of this software are expected to use this software responsibly while abiding the local law. If face of a real person is being used, users are suggested to get consent from the concerned person and clearly mention that it is a deepfake when posting content online. Developers of this software will not be responsible for actions of end-users.
+We are dedicated to advancing this project in a positive direction while adhering to both legal standards and ethics. In certain circumstances, the project may be shut down or watermarks may be added to the output as required by law.
 
+Users of this software are expected to use it responsibly and in accordance with local laws. If a real person's face is used, users are advised to obtain consent from the individual and clearly indicate that the content is a deepfake when sharing it online. The developers of this software will not be held responsible for the actions of end-users.
 
 Documentation
 -------------
```

Comment:
This software aims to be a valuable addition to the AI-generated media industry, assisting artists with tasks like character animation and using characters as clothing models. The developers are mindful of potential ethical issues and have implemented safeguards to prevent misuse. The software includes checks to avoid working on inappropriate content, such as nudity, graphic material, and sensitive subjects like war footage. Responsible use in line with local laws is expected from users, including obtaining consent for real person's faces and labeling deepfake content. The developers prioritize legal and ethical standards, with the possibility of shutting down the project or adding watermarks if required by law. It's important to note that developers won't be held accountable for users' actions.